vipps-ecom-api.md

```diff
@@ -1047,66 +1035,29 @@ See: [Callback statuses](#callback-statuses).
 
 ### Remove User Consent
 
-[`DELETE:[consentRemovalPrefix]/v2/consents/{userId}`](https://vippsas.github.io/vipps-ecom-api/#/Endpoints_required_by_Vipps_from_the_merchant/removeUserConsentUsingDELETE)
+The [`DELETE:[consentRemovalPrefix]/v2/consents/{userId}`](https://vippsas.github.io/vipps-ecom-api/#/Endpoints_required_by_Vipps_from_the_merchant/removeUserConsentUsingDELETE) API call allows Vipps to send an end user's consent removal request to the merchant.
+This endpoint is required for express checkout.
 
-This allows Vipps to send an end user's consent removal request to merchant.
-This endpoint is required for express checkout. When receiving this request,
+When receiving this request,
 the merchant is obliged to handle the user details as per the GDPR guidelines.
 The request path will include a `userId` that Vipps will have provided as
-part of callback, and also made accessible through [`GET:/ecomm/v2/payments/{orderId}/details`](https://vippsas.github.io/vipps-ecom-api/#/Vipps_eCom_API/getPaymentDetailsUsingGET).
-
-## Skip landing page
```

Comment:
Moved "Skip landing page" section down below, right after the "Landing page" section.